bqetl_project.yaml

```diff
@@ -605,3 +608,5 @@ retention_exclusion_list:
 - sql/moz-fx-data-shared-prod/firefox_ios_derived/client_adclicks_history_v1
 - sql/moz-fx-data-shared-prod/acoustic_external/suppression_list_v1
 - sql/moz-fx-data-shared-prod/telemetry_derived/fx_accounts_linked_clients_v1
+- sql/moz-fx-data-shared-prod/firefox_desktop_derived/baseline_clients_city_seen_v1
+- sql/moz-fx-data-shared-prod/fenix_derived/baseline_clients_city_seen_v1
```

Comment:
Does `retention_exclusion_list` support something like `sql/moz-fx-data-shared-prod/**/baseline_clients_city_seen_v1`?  Seems inconvenient to need to add each app